# Constants
# Request
uint8 id
---
# Response
uint32 position